### ISSUE TITLE
Failed to activate the atom-tt package

### ISSUE THREAD

**Issue by pluff** [shemerey/atom-tt#1]
1. install package.
2. select text and press cmd+ctrl+\

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.4
**Thrown From**: [atom-tt](https://github.com/shemerey/atom-tt) package, v0.3.0

### Stack Trace

Failed to activate the atom-tt package

```
At Cannot find module 'space-pen'

Error: Cannot find module 'space-pen'
    at Module._resolveFilename (module.js:338:15)
    at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/shutin/.atom/packages/atom-tt/lib/atom-tt-view.coffee:1:10)
    at Object.<anonymous> (/Users/shutin/.atom/packages/atom-tt/lib/atom-tt-view.coffee:1:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/shutin/.atom/packages/atom-tt/lib/atom-tt.coffee:1:10)
    at Object.<anonymous> (/Users/shutin/.atom/packages/atom-tt/lib/atom-tt.coffee:1:1)
    at Module._compile (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (/Applications/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:715:27)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:173:16)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:795:25
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:20)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:00.9.0 atom-tt:toggle (div.panels-item)
```

### Config

```json
{
  "core": {
    "customFileTypes": {
      "source.ruby": [
        "*.arb"
      ]
    },
    "disabledPackages": [
      "symbols-view",
      "package-cop"
    ],
    "packagesWithKeymapsDisabled": [
      "ruby-test",
      "ruby-block",
      "atom-beautify",
      "linter"
    ],
    "projectHome": "/Users/shutin/www"
  }
}
```

### Installed Packages

```coffee
# User
atom-ctags, v4.5.2
atom-tt, v0.3.0
autocomplete-ruby, v0.1.0
cucumber, v0.5.0
file-icons, v1.7.14
language-diff, v0.4.0
language-haml, v0.24.0
language-rspec, v0.5.0
language-slim, v0.5.0
linter, v1.11.8
linter-rubocop, v0.4.7
linter-slim, v1.0.0
merge-conflicts, v1.4.4
react, v0.15.0
topnet, v0.0.3
tree-view-bundler, v0.0.3

# Dev
No dev packages
```

**Comment by shemerey** [comment]
I've updated dependency, this problem should be gone. Can you check this out with the latest version ?

**Comment by shemerey** [comment]
fixed in `v0.4`
